etc/cvmfs/domain.d/cta-observatory.org.conf

```diff
@@ -0,0 +1,3 @@
+. ../common.conf
+CVMFS_SERVER_URL="http://grid-cvmfs-one.desy.de:8000/cvmfs/@fqrn@;http://cccvmfsctas1.in2p3.fr/cvmfs/@fqrn@"
+CVMFS_KEYS_DIR=$CVMFS_MOUNT_DIR/$CVMFS_CONFIG_REPOSITORY/etc/cvmfs/keys
```

Comment:
This has the same problems as the last PR had; it is missing the directory at the end of the keys and missing the 4 common lines at the beginning.